README.md

```diff
@@ -25,7 +25,7 @@ Read the following chapters for using this application locally:
 
 1. [Create](https://developers.facebook.com/) a new Facebook Application.
 
-2. Install with Composer
+2. Install plugin using composer
```

Comment:
Thanks for suggestion but such application is not a plugin. More like a library. Composer should be also capitalized. Probably in such case more something like:

```
Install application with [Composer](https://getcomposer.org/)
```

in case people aren't familiar with Composer yet, but such cases might be hardly so. Thanks.

Reply:
I agree with @petk.

Reply:
ah i see, as soon as possible  i make new pull request. Thanks for correction

Reply:
great, thanks @jokosu10

Reply:
@jokosu10 you can also send commits to the current `fixgrammar` branch and changes will be visible in this pull request. In case you need some help regarding this let me know.